lola.html

```diff
@@ -896,6 +967,42 @@ <h3>Inferring and Setting Permissions</h3>
         If getting the user intent happens afterward, it probably is best to default to private and then change the content 
         to being more available on user confirmation.</p>
 
+
+      <h3>
+        Load Management During Copying
+      </h3>
+      <p>
+        Copying all the data from an account can be time consuming, as well as eat up resources.  A source server that 
+        announces it is going to shut down is likely to get many transfer requests underway.  The possibility for other 
+        servers to have excess traffic must also be considered.  With the exception of the 429 status code and the need 
```

Comment:
Forgive my ignorance here, but is there any range-request or pagination syntax that a source server needs to be aware of to help destination server request, say, 50 more records starting at record 27, if it is restarting after an error interrupted its import of records 0-49 with only the first 26 succesful? Maybe this is all bog-standard HTTP but even if so, it couldn't hurt to save the people doing this kind of REST work for the first time a little googling 😅